### DISCUSSION TITLE
Installation on Linux Mint 21

### DISCUSSION THREAD

**Discussion by stefan-franz** [hydrogen-music/hydrogen#1676] [Hydrogen on Linux]
Hello, 
how can i install Hydrogen on Linux Mint? 

Thanks in advance - the download page is for me too obscure

**Comment by theGreatWhiteShark** [comment]
Hey @stefan-franz,

On Linux you need to compile it from source. How to do so is described in here https://github.com/hydrogen-music/hydrogen/blob/master/INSTALL.md
